### PR TITLE
bpo-30176: add missing curses cell attributes constants

### PR DESCRIPTION
**Pull request by zhangyangyu**

### CODE REVIEW

File: Doc/library/curses.rst [hunk 1]
```diff
@@ -1271,11 +1271,15 @@ The :mod:`curses` module defines the following data members:
...
    :const:`__version__`.
 
-Several constants are available to specify character cell attributes:
+Some constants are available to specify character cell attributes.
+The exact constants available are system dependent.
 
 +------------------+-------------------------------+
 | Attribute        | Meaning                       |
 +==================+===============================+
+| ``A_ATTRIBUTES`` | Bit-mask to extract           |
```
Comment (serhiy-storchaka) [inline Doc/library/curses.rst]: Either add periods at the end of new descriptions or remove them at the end of existing descriptions.

**Comment by zhangyangyu** [comment]
Now it's grouped and they are in alphabetical order in group.

**Comment by Mariatta** [comment]
Please resolve the conflict. Otherwise I think this is good to go.
Thanks.

**Review by Mariatta — APPROVED** [review]

**Comment by zhangyangyu** [comment]
Thanks @Mariatta ! I am busy recently. I'll do the backports this weekend.

**Comment by serhiy-storchaka** [comment]
RC of 3.6.2 will be released in few hours.

**Comment by zhangyangyu** [comment]
I have to go back home to make PRs. :-( If anyone would like to do the backports then thanks. :-) But I think there will be some conflicts to resolve since `A_ITALIC` doesn't exist in 3.6 and before.

**Comment by Mariatta** [comment]
Bacport PR: #2241. Please review.

**Pull request merged**